docs/usage/automerge-configuration.md

```diff
@@ -138,3 +138,22 @@ These approval helper apps are only available for GitHub.
 ### Codeowners
 
 Depending on the platform, having a `CODEOWNERS` file could block automerging, because it means a code owner must review the PR.
+
+### Disabling automerge in select repos, rest of org should automerge
+
+You might have setup a global configuration in a `.github` repository, that has a `renovate.json` file that turns on automerge for certain dependencies.
+Repositories in the organization all extend from this global configuration, and so they "inherit" the automerge settings as well.
+To turn off automerge in full for select repositories, you need to make a config that overrides all `packageRules` in the repository's `renovate.json` file, like this:
```

Comment:
I'm not sure if this properly explains the use-case for the config below, so please suggest changes to the wording.

Reply:
```suggestion
You might have setup a global configuration in a `.github` repository, that has a `renovate.json` file that turns on automerge for certain dependencies.
Repositories in the organization all extend from this global configuration, and so they "inherit" the automerge settings as well.
To turn off automerge for all dependnecies of a selected repository, you need to make a config that overrides all `packageRules` in the repository's `renovate.json` file, like this:
```

Something like this?

Reply:
I think your text is a bit clearer than mine, so I'll apply your suggestion. 😄